jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ConnectionSettingsMenus.kt

```diff
@@ -167,6 +170,21 @@ open class ProjectLevelSettingSelector(private val project: Project, settingsMod
 }
 
 class ToolkitConnectionComboBoxAction(private val project: Project) : ComboBoxAction(), DumbAware {
+    init {
+        ApplicationManager.getApplication().messageBus.connect().subscribe(
+            BearerTokenProviderListener.TOPIC,
+            object : BearerTokenProviderListener {
+                override fun invalidate(providerId: String) {
+                    val connectionManager = ToolkitConnectionManager.getInstance(project)
```

Comment:
shouldnt this live in DefaultToolkitConnectionManager instead?

Reply:
okk make sense